File: beginner/__3000__/3039/3039.cpp
#include <iostream>

using namespace std;

int main(){

    int n, bonecas = 0, carrinhos = 0;
    cin >> n;

    for(int i = 0; i < n; i++){
        string nome,sexo;
        cin >> nome >> sexo;

        if(sexo == "M"){ carrinhos++; }
        else { bonecas++; }
    }

    cout << carrinhos << " carrinhos" << endl;
    cout << bonecas << " bonecas" << endl;
    
    return 0;
}

File: beginner/__3000__/3039/3039.js
const fs = require("fs")
const input = fs.readFileSync("/dev/std", "utf8").trim().split('\n')

let carrinhos = 0
let bonecas = 0

for(let i = 0; i < input.length; i++){
    let [nome, sexo] = input[i].split(' ')
    if(sexo == "M"){ carrinhos++ }
    else{ bonecas++ }
}

console.log(`${carrinhos} carrinhos`)
console.log(`${bonecas} bonecas`)